content/en/docs/contribute/generate-ref-docs/kubernetes-components.md

```diff
@@ -19,7 +19,7 @@ reference documentation for tools and components in the
 
     * [Python](https://www.python.org/downloads/) v3.7.x
     * [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-    * [Golang](https://golang.org/doc/install) version 1.12 for Kubernetes 1.14+; Go 1.13 [is not supported](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#go)
+    * [Golang](https://golang.org/doc/install) version 1.12 for Kubernetes 1.14+; Go 1.13 for [Kubernetes 1.17+](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#go)
```

Comment:
Hello @everflux ,
Can you simplify line 22 to:
```
* [Golang](https://golang.org/doc/install) version 1.13+
```
Page preview:
https://deploy-preview-18171--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/kubernetes-components/